docs/source/osds/scenarios.rst

```diff
@@ -171,185 +196,204 @@ An example of using the ``non-collocated`` OSD scenario with encryption, bluesto
+provision an OSD are made by the Ceph tooling (``ceph-volume lvm batch`` in
+this case).  There is almost no room to modify how the OSD is composed given an
+input of devices.
 
-   You can not use the same journal for many OSDs.
+To use this configuration, the ``devices`` option must be populated with the
+raw device paths that will be used to provision the OSDs.
 
 
-``filestore``
-^^^^^^^^^^^^^
+.. note:: Raw devices must be "clean", without a gpt partition table, or
+          logical volumes present. Otherwise the tooling will refuse
+          to use them.
+
+For example, for a node that has ``/dev/sda`` and ``/dev/sdb`` intended for
+Ceph usage, the configuration would be:
 
-There is filestore support which can be enabled with:
 
 .. code-block:: yaml
 
-   osd_objectstore: filestore
+   osd_scenario: lvm
+   devices:
+     - /dev/sda
+     - /dev/sdb
 
-To configure this scenario use the ``lvm_volumes`` config option.
-``lvm_volumes``  is a list of dictionaries which expects a volume name and
-a volume group for logical volumes, but can also accept a parition in the case of
-``filestore`` for the ``journal``.
+In the above case, if both devices are spinning drives, 2 OSDs would be
+created, one on each device, with a collocated journal.o
 
-The following keys are accepted for a ``filestore`` deployment:
+Other provisioning strategies are possible, by mixing spinning and solid state
+devices, for example:
 
-* ``data``
-* ``data_vg`` (not required if ``data`` is a raw device or partition)
-* ``journal``
-* ``journal_vg`` (not required if ``journal`` is a partition and not a logical volume)
-* ``crush_device_class`` (optional, sets the crush device class for the OSD)
+.. code-block:: yaml
 
-The ``journal`` key represents the logical volume name or partition that will be used for your OSD journal.
+   osd_scenario: lvm
+   devices:
+     - /dev/sda
+     - /dev/sdb
+     - /dev/nvme0n1
 
-For example, a configuration to use the ``lvm`` osd scenario would look like:
+Similar to the initial example, this would end up producing 2 OSDs, but data
+would be placed on the slower spinning drives (``/dev/sda``, and ``/dev/sdb``)
+and journals would be placed on the faster solid state device ``/dev/nvme0n1``.
 
-.. code-block:: yaml
+Other (optional) supported settings:
 
-   osd_objectstore: filestore
-   osd_scenario: lvm
-   lvm_volumes:
-     - data: data-lv1
-       data_vg: vg1
-       journal: journal-lv1
-       journal_vg: vg2
-       crush_device_class: foo
-     - data: data-lv2
-       journal: /dev/sda
-       data_vg: vg1
-     - data: data-lv3
-       journal: /dev/sdb1
-       data_vg: vg2
-     - data: /dev/sda
-       journal: /dev/sdb1
-     - data: /dev/sda1
-       journal: journal-lv1
-       journal_vg: vg2
+- ``crush_device_class``: Sets the crush class for all OSDs created with this
+  method (it is not possible to have a per-OSD crush class using the *simple*
+  configuration approach)
```

Comment:
Might want to describe, or link to, possible values for this

Reply:
I've added an example, but any string will do here really.